index.bs

```diff
@@ -4532,39 +4735,31 @@ partial interface MLGraphBuilder {
+
+    : <dfn>keepDimensions</dfn>
+    ::
+        A {{boolean}}. If true, retains reduced dimensions with [=list/size=] 1. The default value is false.
+</dl>
+
 <div>
     **Arguments:**
         - *input*: an {{MLOperand}}. The input tensor.
         - *options*: an optional {{MLReduceOptions}}. The optional parameters of the operation.
-            - *axes*: a sequence of {{unsigned long}}. The dimensions to reduce. The values in the sequence must be in the range [0, N-1] where N is the [=rank=] of the input tensor.
-                If not present, all dimensions are reduced.
-            - *keepDimensions*: a {{boolean}}. If true, retains reduced dimensions with [=list/size=] 1.
-                The default value is false.
 
     **Returns:** an {{MLOperand}}. The reduced output tensor.
 </div>
 
-<div class="note">
-    **Reduction types:**
-        - *L1*: Compute the <a href="https://mathworld.wolfram.com/L1-Norm.html">L1 norm</a> of all the input values along the axes.
-        - *L2*: Compute the <a href="https://mathworld.wolfram.com/L2-Norm.html">L2 norm</a> of all the input values along the axes.
-        - *LogSum*: Compute the log value of the sum of all the input values along the axes.
-        - *LogSumExp*: Compute the log value of the sum of the exponent of all the input values along the axes.
-        - *Max*: Compute the maximum value of all the input values along the axes.
-        - *Mean*: Compute the average value of all the input values along the axes.
-        - *Min*: Compute the minimum value of all the input values along the axes.
-        - *Product*: Compute the product of all the input values along the axes.
-        - *Sum*: Compute the sum of all the input values along the axes.
-        - *SumSquare*: Compute the sum of the square of all the input values along the axes.
-</div>
-
 <details open algorithm>
-
   <summary>
     To <dfn for="MLGraphBuilder" data-lt="reduce-op">create reduce operation</dfn> given |op|, |input| and |options|, run the following steps:
   </summary>
   <div class=algorithm-steps>
-    1. [=Assert=]: |op| is one of "reduceL1", "reduceL2", "reduceLogSum", "reduceLogSumExp", "reduceMax", "reduceMean", "reduceMin", "reduceProduct", "reduceSum", "reduceSumSquare".
+    1. [=Assert=]: |op| is one of "reduceArgMax", "reduceArgMin", "reduceL1", "reduceL2", "reduceLogSum", "reduceLogSumExp", "reduceMax", "reduceMean", "reduceMin", "reduceProduct", "reduceSum", "reduceSumSquare".
     1. [=Assert=]: the type of |input| is {{MLOperand}}.
     1. If |options|.{{MLReduceOptions/axes}} [=map/exists=], if any of its elements is not in [=the range=] 0 to the [=rank=] of |input|, exclusive, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
     1. If any of the following sub-steps fail, [=exception/throw=] an "{{OperationError}}" {{DOMException}}.
```

Comment:
There seems to be two issues of `output` operand creation:
1. The output shape should be calculated instead of just copying input's.
2. The output data type of `reduceArgMax` and `reduceArgMin` should be unsigned integer rather than setting to input's.

Reply:
I did not change the existing wording here besides adding the new argMin/Max. Please suggest the wording to add.

Reply:
I'd recommend to make standalone `argMax` and `argMin` operators in another [comment](https://github.com/webmachinelearning/webnn/pull/478#discussion_r1405292717).

Reply:
@wchao1115 suggested we work this out in a separate issue in https://github.com/webmachinelearning/webnn/pull/478#discussion_r1405292717 but on our WG call https://www.w3.org/2023/11/30-webmachinelearning-minutes.html#t07 the sentiment was to fix this in this branch.

Does the spec remain in a cohesive state if we spin this into a separate issue?

As agreed, the general options are to either spin into a separate issue or resolve in this PR.

Reply:
Fixed.